### ISSUE TITLE
Collapse the node context to show the node based on its sythn.

### ISSUE THREAD

**Issue by eznix86** [wire-elements/wire-spy#3]
Currently these are the definition of the synth.

form -> Form,
str -> Stringable,
elcln -> Eloquent Collection,
elmdl -> Eloquent Model (Legacy),
int -> Int,
arr -> Array,
clctn -> Collection,
elcl -> Eloquent Collection (Legacy),
float -> Float,
enm -> Enum,
cbn -> Carbon,
fil -> File,
std -> StdClass,
mdl -> Model,
wrbl -> Wireable

**Comment by PhiloNL** [comment]
Hi @eznix86 
Can you provide some more context on this, not sure I fully understand.

**Comment by eznix86** [comment]
It is a quick example:

<img width="379" alt="Screenshot 2024-10-12 at 19 51 21" src="https://github.com/user-attachments/assets/02ea9cb8-fa90-4027-80e4-b2a94536ee3c">

Livewire sends data in a hydrated form, which Caleb calls Synths, this is hydrated into a json in the frontend, when it is sent back to the server, we can translate it back to what is was "originally" was.

Ex. Have an Eloquent Class, it would be better, to show this Eloquent Class/Model with an icons to present what it is, maybe if we click on it we see the attributes, a bit like a `die dump` of the data.

Basically instead of expanding known synth (the list above), we can display them in a way where it make sense for a dev.
